.buildkite/commands/build-for-testing.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash -eu
 
+# Temporary fix until we're on the Xcode 13.1 VM
```

Comment:
I fear that since those comments are hidden in the separate `.sh` files and thus not noticable directly from the `.yml` pipelines, we might miss them next time we update the image.

What do you think about doing something like this:

1. Add some hashtag to this comment, like:
```suggestion
# FIXIT-13.1: Temporary fix until we're on the Xcode 13.1 VM
```
2. Add a reference to this comment in the 2 `.yml` files:
```yml
    IMAGE_ID: xcode-13 # Remember to address all 'FIXIT-13.1' comments when we'll update to new image
```